src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java

```diff
@@ -419,14 +424,18 @@ public boolean onAreaTouched(final TouchEvent pSceneTouchEvent,
         String playerStr = "Played by " + stat.getPlayerName() + " on " +
                 new java.text.SimpleDateFormat("yyyy/MM/dd HH:mm:ss", Locale.getDefault()).format(new java.util.Date(stat.getTime()));
         playerStr += String.format("  %s(%s)", BuildConfig.VERSION_NAME, BuildConfig.BUILD_TYPE);
-        if (stat.getChangeSpeed() != 1 || stat.isEnableForceAR()){
+
+        if (stat.getChangeSpeed() != 1 || stat.isEnableForceAR() || Float.compare(stat.getFLFollowDelay(), FlashLightEntity.defaultMoveDelayS) != 0){
```

Comment:
Not sure if this is the correct direction since the follow delay is already stored in replay, so it should be reflected properly when watching replays.

It's not like it will break past pre-releases due to the way this is stored in replay.

Reply:
ah, ok, I think this should only be shown if FL is active? Otherwise it wouldn't make sense (submitted replay has custom FL follow delay, but FL isn't active).

Reply:
Oh yeah